### PR TITLE
Update designing.mdx

### PR DESCRIPTION
**Pull request by oliviaflory**
Updates:

– Added in Carbon Expressive theme design kits

– Removed resource card to request access through Github
– Added instructions to request access through Design kit channel on slack

– Fixed instruction paths to design kit to follow the IBM design kit parent folder on Box

### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by jacobottesen-dgc** [comment]
@wonilsuhibm @JennySanchez What are your thoughts on adding a resource card for the request access section?
Screen Shot 2020-10-13 at 3 54 19 PM

**Comment by JennySanchez** [comment]
@jacobottesen-dgc We have it currently on the [website](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/get-started/designing). We used the Resource Card treatment bc it takes the adopter directly to the Access-Request Git template for the adopter to fill in the information. Right now, since we are moving the design kits to the IBM design kit folder, I think what she has is clear and enough.  Without the Resource card in between the steps is actually easier for me to follow the steps. What do you think?

File: src/pages/get-started/designing.mdx [hunk 1]
```diff
@@ -23,45 +23,35 @@ Rapidly build beautiful and accessible experiences. The Carbon for IBM.com kit a
...
 
-The Carbon for IBM.com kit includes expressive type styles for editorial experiences and IBM.com specific components including the IBM.com masthead and footer. Teams should utilize the Carbon design kits for core components such as buttons, form components, etc. If you have any questions please reach out on our [slack channel](https://cognitive-app.slack.com/archives/C2PLX8GQ6).
+The Carbon for IBM.com and Carbon Expressive kits includes expressive type styles for editorial experiences and IBM.com specific components including the IBM.com masthead and footer. Teams should utilize the Carbon Expressive design kits for core components such as buttons, form components, etc. If you have any questions please reach out on our [slack channel](https://cognitive-app.slack.com/archives/C2PLX8GQ6).
 
 ## Get the kit
 
 **1. Install Sketch.**
-To design with the Carbon for IBM.com kit you must have the most recent version of [Sketch](https://www.sketch.com) installed.
+To design with the Carbon for IBM.com kit and Carbon Expreessive kit you must have the most recent version of [Sketch](https://www.sketch.com) installed.
```
Comment (oliviaflory) [inline src/pages/get-started/designing.mdx:31]:
```suggestion
To design with the Carbon for IBM.com kit and Carbon Expressive kit you must have the most recent version of [Sketch](https://www.sketch.com) installed.
```

File: src/pages/get-started/designing.mdx [hunk 2]
```diff
@@ -23,45 +23,35 @@ Rapidly build beautiful and accessible experiences. The Carbon for IBM.com kit a
...
+**4. Set up Sketch library and templates.** 
+
+You can save the template and library by navigating to the `IBM Design kit / Digital design / Carbon for IBM.com design kit` folder.
 
 To save the grid template, open the “IBM Design template” file in Sketch, from the menu choose `File → save as template → save`.
 
-Follow the same instructions for the "IBM.com Shell template".
 
-To add the libraries, from Sketch navigate to `Preferences → Libraries → Add Library` and select `Box →  Carbon for IBM.com design kit → Carbon for IBM.com (White theme)`.
+To add the libraries, from Sketch navigate to `Preferences → Libraries → Add Library` and select `Box → Digital design → IBM Design kit → Carbon for IBM.com → Carbon Expressive (White theme)`.
```
Comment (oliviaflory) [inline src/pages/get-started/designing.mdx:48]:
```suggestion
To add the libraries, from Sketch navigate to `Preferences → Libraries → Add Library` and select `Box → IBM Design kit → Digital design → Carbon for IBM.com → Carbon Expressive (White theme)`.
```

**Review by jacobottesen-dgc — APPROVED** [review]

**Review by JennySanchez — APPROVED** [review]

**Pull request merged**